README.md

```diff
@@ -87,11 +89,27 @@ import { AppComponent } from './app.component';
     BrowserModule
   ],
   providers: [],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  bootstrap: [AppComponent]
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+  bootstrap: [ AppComponent ]
 })
 export class AppModule { }
 
+// or the vanilla JS version
+
+(function(app) {
+  app.AppModule =
+    ng.core.NgModule({
+      imports: [ ng.platformBrowser.BrowserModule ],
+      declarations: [ app.AppComponent ],
+      schemas: [ ng.core.CUSTOM_ELEMENTS_SCHEMA ],
+      bootstrap: [ app.AppComponent ]
+    })
+    .Class({
+      constructor: function() {}
+    });
+})(window.app || (window.app = {}));
+
+
 /*
 You can now use the custom element in either a standalone or inline template
```

Comment:
Yeah, the cool thin about Angular us that once you've added the CUSTOM_ELEMENTS schema that's it. The only thing you need to do is use your custom element as a regular HTML tag in your angular template as stated in the last part of the example:

<custom-widget [attr.title]="title"></custom-widget>